src/commands/pod_install.yml

```
@@ -25,7 +29,7 @@ steps:
   - run:
       name: Install CocoaPods
       command: |
-        cd <<parameters.pod_install_directory>> && pod install && cd -
+        cd <<parameters.pod_install_directory>> && <<parameters.pod_install_command>> && cd -
```

@fotos even though this worked with my alternate command (`bundle exec pod install`) i'm wondering if it would be preferable to use `eval` in case another use-case requires it

> The eval command is useful when executing a command that contains a special operator or reserved keywords or in scripts with variables whose names are unknown until the script executes

```suggestion
        cd <<parameters.pod_install_directory>> && eval <<parameters.pod_install_command>> && cd -
```

---

The [node orb uses eval](https://github.com/CircleCI-Public/node-orb/blob/66a75a3bdb20f859a7dd807f9023d17f0a94fdc9/src/scripts/packages/yarn-v1-install.sh#L3), so it feels like a best practice. I can update it Monday so please feel free to hold merging until then

---

@bhandanyan-nomad `eval` would make sense when the parameter contains special characters or is a script. I'm not sure which use-case would need that. By the way the ruby orb is using `eval` only in [one case](https://github.com/CircleCI-Public/ruby-orb/blob/0f96239868e5ac3aad51cd782900510b66faa5a3/src/commands/install.yml#L4-L8).

In any case feel free to open the PR with `eval` if you find it useful/reasonable.

---

I don't think it hurts, so I've gone ahead and added it